ci/run_tests.sh

```diff
@@ -19,6 +19,8 @@ exit_suite --continue
 
 enter_suite tests
 
+export TREE_SITTER_DIR=$HOME/tree-sitter-build/
```

Comment:
```suggestion
export TREESITTER_DIR=$HOME/tree-sitter-build/
```

Reply:
`TREE_SITTER_DIR` is used by the tree-sitter binary, it is not a name chosen by us.